### DISCUSSION TITLE
How to show an image in-line with text?

### DISCUSSION THREAD

**Discussion by arj0019** [typst/typst#1119] [Q&A] [answered]
I am creating a document that will use ditto syntax ( " ) to indicate repeated information.

As a joke, I would like to replace the standard ditto double quote ( " ) with a picture of the Pokémon  [ditto](https://img.gamewith.net/article_tools/pokemon-lets-go/gacha/1112.png).

This code almost works, but it creates a new line which prevents me from using it in text:

`#show "ditto": image("ditto.png", height: 12pt, fit: "contain")`

I thought I remembered an example of this functionality somewhere, but I can not find it.

**Comment by Enivex** [comment, marked as answer]
Wrap it in a box

**Reply by arj0019** [reply]
Box achieves the desired functionality.

`#show "ditto": box(image("ditto.png", height: 12pt, fit: "contain"), baseline: 2pt)`
